components/modal/style.scss

```diff
@@ -0,0 +1,105 @@
+.edit-post-plugin-modal__editor-modal {
```

Comment:
The prefix should be `.components-modal` according to the coding standards.
You could use the following structure for better readability:
```
.components-modal {
    &__screen-overlay {}
    &__content {}
    &__header {}
}
```